packages/fx-core/src/component/resource/botService/botRegistration/botRegistration.ts

```diff
@@ -30,6 +30,7 @@ export class Constants {
   public static readonly BOT_REGISTRATION: string = "BotRegistration";
   public static readonly CREATE_BOT_REGISTRATION: string = "createBotRegistration";
   public static readonly UPDATE_MESSAGE_ENDPOINT: string = "updateMessageEndpoint";
+  public static readonly UPDATE_BOT_REGISTRATION: string = "updateBotRegistration";
```

Comment:
Given we will provide updateBotRegistration() for updating more than message endpoint, do we still need to keep the original updateMessageEndpoint()?

Reply:
It is still referenced by legacy code in some `configure` lifecycle of `botService.ts`. We may delete this interface if we're ready to remove all those legacy code.